lib/ProductOpener/Ingredients.pm

```diff
@@ -3757,7 +3757,11 @@ sub preparse_ingredients_text($$) {
 
 	$log->debug("preparse_ingredients_text - before language specific preparsing", { text => $text }) if $log->is_debug();
 
-	if ($product_lc eq 'es') {
+	if ($product_lc eq 'de') {
+		# deletes comma in "Bienenwachs, weiß und gelb" since it is just one ingredient
+		$text =~ s/Bienenwachs, weiß und gelb/Bienenwachs weiß und gelb /ig;
```

Comment:
It should work. It will insert an extra space after "gelb" though.

Reply:
Okay fix is on the way